frontend/test/lit/test_decomposition.py

```diff
@@ -115,7 +115,7 @@ def decompose_multicontrolled_x1(theta: float):
         # CHECK:     quantum.custom "PauliX"() {{%[a-zA-Z0-9_]+}} ctrls({{%[a-zA-Z0-9_]+}}, {{%[a-zA-Z0-9_]+}}, {{%[a-zA-Z0-9_]+}})
         # CHECK-NOT: name = "MultiControlledX"
         qml.MultiControlledX(wires=[0, 1, 2, 3])
-        return qml.state()
+        return qml.expval(qml.X(0))
```

Comment:
Was state not working here? (I don't see shots being used)

Reply:
Thanks, good catch! Looks like this was a test I "fixed" before I realized a lot of the failing state measurements were because of the bug with `ProgramFeatures`. I've put it back now :)